tests/integration/distribution.go

```diff
@@ -137,7 +137,16 @@ func (s *CCVTestSuite) TestRewardsDistribution() {
 	}
 	consuValsRewards := consumerValsOutstandingRewardsFunc(s.providerCtx())
 
-	// Transfer rewards from consumer to provider
+	// increase the block height so validators are eligible for consumer rewards (see `IsEligibleForConsumerRewards`)
```

Comment:
from where is this change comming from?
did not see this change in the original PR 1929.
can you pls cross check this?